Tests/LibC/TestLibCInodeWatcher.cpp

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <Kernel/API/InodeWatcherEvent.h>
+#include <Kernel/API/InodeWatcherFlags.h>
+#include <LibTest/TestCase.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <utime.h>
+
+constexpr unsigned maximum_event_size = sizeof(InodeWatcherEvent) + NAME_MAX + 1;
+u8 buffer[maximum_event_size];
```

Comment:
A struct containing a `InodeWatcherEvent`, and then a trailing array of char/u8  the size of `NAME_MAX + 1` might be a bit nicer to read...